apps/backend/src/models/session.py

```
@@ -0,0 +1,101 @@
+from src.models.base import BaseModel
+
+
+class Session(BaseModel):
+    """Session model for tracking user sessions and JWT tokens."""
+
+    __tablename__ = "sessions"
+
+    # User relationship
+    user_id = Column(Integer, ForeignKey("users.id", ondelete="CASCADE"), nullable=False)
+    user = relationship("User", back_populates="sessions")
+    
```

Ruff (0.11.9)

Blank line contains whitespace

Remove whitespace from blank line

(W293)